products/tarantool.md

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
Is it worth to add a `releaseImage` property (I found it in the [contributing guide](https://endoflife.date/contribute)) with the image from https://www.tarantool.io/en/doc/latest/release/release-calendar/ ?

Reply:
Added. (It was not clear for me what does "release image" mean)